src/morale.h

```diff
@@ -76,6 +73,61 @@ enum morale_type : int {
     NUM_MORALE_TYPES
 };
 
+// Morale multiplier
```

Comment:
Since this struct is never used outside the morale code (and is unlikely to be used elsewhere in the future), could you move its definition to morale.cpp?